docker-compose-tests.yaml

```diff
@@ -27,10 +27,11 @@ services:
     volumes:
       - ./database-service:/app
       - ./fixtures:/app/fixtures
+      - ./test-utils:/app/tests/utils
```

Comment:
There is code that we need to share across multiple services. For example, here, some of the test utils.